app/views/organisations/schemes.html.erb

```diff
@@ -22,11 +22,12 @@
 <% end %>
 
 <div class="app-filter-layout" data-controller="filter-layout">
-  <% if SchemePolicy.new(current_user, nil).create? %>
-    <div class="govuk-button-group app-filter-toggle govuk-!-margin-bottom-6">
-      <%= govuk_button_link_to "Create a new supported housing scheme", new_scheme_path, html: { method: :post } %>
-    </div>
-  <% end %>
+  <% display_actions = SchemePolicy.new(current_user, nil).create? %>
+  <div class="govuk-button-group app-filter-toggle <%= "govuk-!-margin-bottom-6" if display_actions %>">
+    <% if display_actions %>
+        <%= govuk_button_link_to "Create a new supported housing scheme", new_scheme_path, html: { method: :post } %>
+    <% end %>
+  </div>
```

Comment:
The variable `display_actions` here and elsewhere allows us to use the same logic to conditionally show the action buttons and add the bottom margin.